src/test/resources/application.properties

```diff
@@ -39,10 +31,10 @@ docs.trustedS3bucketName=trusted-bucket
 docs.trustedS3bucket=${docs.trustedS3bucketName}
 docs.trustedS3bucketKMSKeyId=123456
 
-clamav.root=http4://localhost:8086
+clamav.root=http4://localhost:9002
 clamav.path=${clamav.root}/scan
 
-hocsconverter.root=http4://localhost:8084
+hocsconverter.root=http4://localhost:9002
```

Comment:
This also runs at port 8084, makes me wonder if that test profile was actually used anywhere.

Reply:
This is to use the wiremock port in the integration tests